lib/config/migrations/custom/package-rules-migration.spec.ts

```diff
@@ -57,4 +57,85 @@ describe('config/migrations/custom/package-rules-migration', () => {
       }
     );
   });
+
+  it('should migrate single docker language tag', () => {
+    expect(PackageRulesMigration).toMigrate(
+      {
+        packageRules: [
+          {
+            matchLanguages: ['docker'],
+            addLabels: ['docker'],
+          },
+        ],
+      },
+      {
+        packageRules: [
+          {
+            matchManagers: [
```

Comment:
why did you prefer to map this to matchManagers instead of matchCategories? I'm ok with making this breaking in order to get it simpler

Reply:
The old `docker` language is not the same as the new `docker` category. 

The language included for example references like `gitlabci`. 
I can remove the migration code, if you prefer that.